packages/@okta/vuepress-theme-prose/components/Home.redesign.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="redesign-homepage container-fluid">
+    <div class="row">
+      <div
+        class="col-xl-2 col-lg-2 d-none d-lg-block redesign-homepage--col-no-padding"
+      >
+        <RedesignedSideBar />
+      </div>
+      <div class="col-xl-10 col-lg-10 col-md-12 col-sm-12">
+        <div class="row justify-content-center">
+          <div class="col-11 redesign-homepage--elevated">
+            <div class="row">
+              <div class="col-12">
+                <h1
+                  class="redesign-homepage--main-heading redesign-homepage--color-main"
+                >
+                  Okta Developer Portal
+                </h1>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-12">
+                <h2
+                  class="redesign-homepage--sub-heading redesign-homepage--color-main"
+                >
+                  Let's get started
+                </h2>
+              </div>
+            </div>
+            <div class="row">
+              <div class="col-12">
+                <p
+                  class="redesign-homepage--sub-heading-disclaimer redesign-homepage--color-sub"
+                >
+                  Authenticate up to 1,000 users per month for free forever
+                </p>
+              </div>
+            </div>
+            <div class="row redesign-homepage--selector-block-margin ">
+              <div
+                class="col-xl-3 col-lg-6 col-md-6"
+                v-for="(tile, index) in $page.frontmatter.tiles"
+                :key="index"
```

Comment:
I've been curious if there is a functional difference between `v-bind:key` and `:key`?

Reply:
Comes down to preference I think: https://stackoverflow.com/a/55105206

Reply:
Looks like `:` is the shorthand for `v-`